book_maker/loader/epub_loader.py

```
@@ -534,11 +877,80 @@ def make_bilingual_book(self):
+
+                if self.accumulated_num > 1:
+                    print(
+                        f"📝 Each chapter applies accumulated_num={self.accumulated_num} independently"
+                    )
+
+                if self.context_flag:
+                    print(
+                        f"🔗 Context enabled: each chapter maintains independent context (limit={self.translate_model.context_paragraph_limit})"
+                    )
+                else:
+                    print(f"🚫 Context disabled for this translation")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)